src/MegaPoker.t.sol

```diff
@@ -68,7 +68,7 @@ interface Hevm {
 }
 
 contract MegaPokerTest is DSTest, PokingAddresses {
-    SpellLike    constant spell     = SpellLike(0xc0Dc70048F678e0c9DA82aD11CdC063aA081d950);
+    SpellLike    constant spell     = SpellLike(0xcdC89bF6EfB7EEd5Ba0Ded0fC9f134b939faFBB0);
```

Comment:
This is actually not needed as the spell was already executed.

Reply:
Done in 907c9b8.